src/bin/wallet.rs

```diff
@@ -198,14 +198,14 @@ fn parse_args() -> Result<WalletConfig, Box<error::Error>> {
                 let sig = Signature::new(&sig_vec);
                 Ok(WalletCommand::Confirm(sig))
             } else {
-                display_actions();
+                println!("{}", matches.usage());
                 Err(WalletError::BadParameter("Invalid signature".to_string()))
             }
         }
         ("balance", Some(_balance_matches)) => Ok(WalletCommand::Balance),
         ("address", Some(_address_matches)) => Ok(WalletCommand::Address),
         ("", None) => {
-            display_actions();
+            println!("{}", matches.usage());
```

Comment:
Hmm. I am dissatisfied with `matches.usage()` here, as it doesn't print the possible subcommands. I don't see an immediate way to get clap to print out the help text instead, but that would be my preference. I'll look into it briefly. That said, there are plenty of CLIs out there that do it this way, so not a big deal. I just think it's not the nicest UX.

Reply:
yeah I guess you add a `-h` or `--help` to the command-line when you arrive at this message and need more help :)

Reply:
Yep, just an extra step.